packages/ui/components/input-stepper/src/LionInputStepper.js

```diff
@@ -84,6 +97,7 @@ export class LionInputStepper extends LocalizeMixin(LionInput) {
 
     this.__increment = this.__increment.bind(this);
     this.__decrement = this.__decrement.bind(this);
+    this.__boundOnEnterButton = this._onEnterButton.bind(this);
```

Comment:
Maybe we can bind the same name under this like we do in all components? (and align this behavior throughout the whole comp (and later write a linting rule for this :) ))?